src/createGridComponent.js

```diff
@@ -364,12 +364,17 @@ export default function createGridComponent({
         // So we need to determine which browser behavior we're dealing with, and mimic it.
         const outerRef = ((this._outerRef: any): HTMLElement);
         if (direction === 'rtl') {
-          const isNegative = isRTLOffsetNegative();
-          if (isNegative) {
-            outerRef.scrollLeft = -scrollLeft;
-          } else {
-            const { clientWidth, scrollWidth } = outerRef;
-            outerRef.scrollLeft = scrollWidth - clientWidth - scrollLeft;
+          switch (getRTLOffsetType()) {
+            case 'negative':
+              outerRef.scrollLeft = -scrollLeft;
+              break;
+            case 'reverse':
```

Comment:
These labels are confusing to me. Especially calling a non-spec-compliant behavior "default". I wonder if there is a more meaningful name we could pick?

So I think we're differentiating between:
* spec-compliant browsers: 0 ... -N
* Chrome: N ... 0
* non-Chromium Edge: 0 ... N

Maybe the following labels would be a bit more meaningful?
* negative: (0 ... -N)
* positive-descending: (N ... 0)
* positive-ascending: (0 ... N)